xml/art-obs-beginners-guide.xml

```diff
@@ -1130,23 +1218,22 @@ Patch0:         &gitproject;_main.diff</screen>
  <sect1 xml:id="sec.obsbg.uc.install-from-obs">
   <title>Installing Packages from &obsa;</title>
   <para>
-   &obsa; provides a place which contains all the distributions and architectures
-   of your successfully built packages. When you create a package
+   &obsa; provides a place which contains all the distribution-specific and
+   architecture-specific versions
+   of successfully built packages. When you create a package
    in your &obsa; home project, all successfully built packages will appear
    under the <uri>&obsdnlurlhome1;</uri> URL.
   </para>
   <para>
+   However, this only true for the home project itself and manually created
+   subprojects but not for subprojects created as a result of branching a
+   package. Branched projects are not published by default.
```

Comment:
correct, but could you point out that developers can still manually download the build results (osc getbinaries). They just don't get a repository that way.